### ISSUE TITLE
Change recommended tool to Postman and add reference to repository with API examples 

### ISSUE THREAD

**Issue by aleksander-dytko** [camunda/camunda-docs#1438]
## Context 
https://camunda.slack.com/archives/C9B5270DA/p1666886067731129

## Expected solution
- Change linked tool for GraphQl to Postman 
- Link [Postman repository](https://github.com/camunda-community-hub/camunda-8-api-postman-collection) in Operate, Tasklist and Console API documentation 
- Add link to [Postman workspace](https://www.postman.com/restless-shuttle-19836/workspace/camunda-platform-8/)

**Comment by akeller** [comment]
I LOVE this! And I'm excited to see some care going into the Postman workspace. @pepopowitz and I have talked about this and our API experience in general.

Is this something the component teams will take care of? Looks like it's on the Operate Board currently. 

Do you want @christinaausley to review any of the content on the Postman workspace itself? She's looking for more opportunities to work closely with API documentation.

**Comment by aleksander-dytko** [comment]
@akeller great to hear that 🤟 

I included it in Operate backlog as this is directly connected with Help request from customer / support team.

From the long-term perspective, I should dicuss who will take care of updating github and postman repository for that as I saw that other PM also include this in their user stories e.g. [here](https://github.com/camunda/product-hub/issues/448). 

I think it would be awesome, if @christinaausley can review both repos - I created a ticket [here](https://github.com/camunda/camunda-platform-docs/issues/1457). 

@akeller thanks for picking that up 🚀
